### PR TITLE
Initial files for new variables interface

### PR DESCRIPTION
**Pull request by rorbol**

### CODE REVIEW

**Comment by rorbol** [comment]
Backup & security tooling still pending

File: output.tf [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+}
+
+output "enabled_regions" {
+  value = var.config.enabled_regions
+}
+
+output "break_glass_access" {
+  value = [for username in var.config.break_glass_accounts : {
+    username = username,
+    password = aws_iam_user_login_profile.break_glass_access[username].password
```
Comment (jesstruck) [inline output.tf:12]: Should we not store this as a SSM parameter instead??

File: variables.tf [hunk 2]
```diff
@@ -1,57 +1,110 @@
...
+  # }
+
+  # validation {
+  #   error_message = ""
+  #   condition = 
+  # }
+  # validation {
+  #   error_message = ""
+  #   condition = 
+  # }
```
Comment (jesstruck) [inline variables.tf:109]: do we need this?

**Review by jesstruck — CHANGES_REQUESTED** [review]

File: account_log_archive.tf [hunk 3]
```diff
@@ -0,0 +1,182 @@
...
+  statement {
+    sid       = "AllowAssumeRole"
+    actions   = ["sts:AssumeRole"]
+    resources = [format(local.organization_role_arn_template, local.log_archive_account_id)]
+
+    # condition {
+    #   test     = "DateLessThan"
+    #   variable = "aws:CurrentTime"
+    #   values   = [timeadd(timestamp(), "30m")]
+    # }
```
[the remark this answers is not in the document]
Comment (rorbol) [reply account_log_archive.tf:15]: fixed

File: variables.tf (hunk 2 above)
Comment (rorbol) [reply variables.tf:109]: Yes, we do need validation of inputs :)

File: output.tf (hunk 1 above)
Comment (rorbol) [reply output.tf:12]: No - this is by design. Break glass access is intended for when there is no other access to the account including ssm.

**Pull request merged**